docs/modules/ROOT/pages/how-tos/testing/integration/periodic-integration-tests.adoc

```diff
@@ -0,0 +1,134 @@
+        spec:
+          containers:
+            - name: trigger-e2e-scenario
+              image: 'quay.io/konflux-ci/appstudio-utils:latest'
+              imagePullPolicy: Always
+              command:
+                - /bin/bash
+                - '-c'
+              args:
+                - |
+                  #!/bin/bash
+                  set -euo pipefail
+
+                  export KONFLUX_SCENARIO_NAME="integration-test" <.>
+                  export KONFLUX_TENANT_NAME="default-tenant" <.>
+                  export KONFLUX_APPLICATION_NAME="application-default" <.>
+                  export KONFLUX_COMPONENT_NAME="component-default" <.>
+
+                  echo -e "[INFO] Fetching latest snapshot from ${KONFLUX_TENANT_NAME} related to push events."
+
+                  LATEST_SNAPSHOT=$(kubectl get snapshots -n "${KONFLUX_TENANT_NAME}" -o json | \
+                      jq --arg application "$KONFLUX_APPLICATION_NAME" --arg component "$KONFLUX_COMPONENT_NAME" -r '
+                          .items
+                          | map(select(
+                              .metadata.labels."appstudio.openshift.io/application" == $application and
+                              .metadata.labels."appstudio.openshift.io/component" == $component and
+                              .metadata.labels."pac.test.appstudio.openshift.io/event-type" == "push" and
+                              (.status.conditions // [] | map(select(
+                                  .type == "AutoReleased" and
+                                  .reason == "AutoReleased" and
+                                  .status == "True"
+                                  ))
+                              | length > 0)
+                              ))
+                          | sort_by(.metadata.creationTimestamp) | last | .metadata.name')
+
+                  if [[ -z "${LATEST_SNAPSHOT}" || "${LATEST_SNAPSHOT}" == "null" ]]; then
+                    echo -e "[ERROR] No valid snapshot found. The job will not be triggered."
+                    exit 1
+                  fi
+
+                  echo -e "[INFO] Triggering test scenario ${KONFLUX_SCENARIO_NAME} from snapshot ${LATEST_SNAPSHOT}."
+
+                  kubectl -n "${KONFLUX_TENANT_NAME}" label snapshot "${LATEST_SNAPSHOT}" test.appstudio.openshift.io/run="${KONFLUX_SCENARIO_NAME}"
+
+                  echo "[INFO] Integration Service E2E tests successfully triggered!"
+          serviceAccountName: default
+          serviceAccount: default
+----
+<.> The name of the tenant.
+<.> The Integration Test name to be triggered by the CronJob.
+<.> The tenant from where to annotate the component snapshot to trigger Periodic Integration Test.
+<.> The {ProductName} application name associated with the Integration Test that will be triggered by the CronJob.
+<.> The {ProductName} componet name.
+
+Once the CronJob is triggered you should see your integration tests pipelines executing in {ProductName} UI.
```

Comment:
You will see it immediately? Or only after some time (i.e. after the cron has triggered)?